### PR TITLE
Re-structuring files so a package can be uploaded to PyPi

### PR DESCRIPTION
**Pull request by edmoss345**

### CODE REVIEW

**Comment by edmoss345** [comment]
@istride this is the repo which contains a python function we needed to automate Chiara's RapidPro flow creating process. Not particularly clean, but everything that relates to the package is contained within the 'src' folder

File: main.py [hunk 1]
```diff
@@ -49,4 +49,4 @@ def main():
     
 
 if __name__ == '__main__':
-    main()
+    main()
```
Comment (istride) [inline main.py:52]: This looks like an unnecessary, accidental, whitespace-only change. Please avoid this as it clutters the diff and causes confusion.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[metadata]
+description-file=README.md
+license_files=LICENSE.rst
```
Comment (istride) [inline setup.cfg:3]: There is an existing license file, called 'LICENSE'.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='rapidpro_flow_tools',
+    version='2.0.3',
+    license='MIT',
```
Comment (istride) [inline setup.py:6]: Use the license of the project.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    author='Ed Moss',
+    author_email='ed.moss@eemengineering.com',
```
Comment (istride) [inline setup.py:8]: IDEMS International
communications@idems.international

**Review by istride — CHANGES_REQUESTED** [review]
Struggling to understand what is happening here as it's difficult to tell the difference between what is new and what is old and has merely been moved.

Are we duplicating source files in an attempt to accommodate two different ways of using this project i.e. with and without packaging? If so we should find a way to manage the transition better. The current method of duplication will break the git history for those files, which is almost the whole project. A better way might be to maintain two branches while we transition to the packaged version.

I don't understand why there are various 'models' modules  under 'test', for epicsa, parenttext and srh_chatbot - this smells, to me.

**Comment by edmoss345** [comment]
Yup sorry @istride this one is a bit of a mess. I forgot that I started with a branch of Chira's so the content related to the models should be broken out.

Essentially I have put everything needed for the package in the 'src' directory as I didn't want to break it for anyone who is not using the packaging. Yes the vast majority of things have just been copied into 'src' to be included in the package, although I did have to make lots of small changes to the internal referencing within the subpackages so that everything looks in the right place.

Your suggestion of keeping in an independent branch makes the most sense so will do that and then we will still have the git history

File: setup.cfg (hunk 2 above)
Comment (edmoss345) [reply setup.cfg:3]: have pointed to LICENSE file as suggested

File: setup.py (hunk 3 above)
Comment (edmoss345) [reply setup.py:6]: changed

**Pull request closed without merging**